infra/frontend/service/main.tf

```diff
@@ -1,12 +1,10 @@
-# docs: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/vpc
```

Comment:
I used to add these comments, mostly for myself, but then I found a VS code extension that provides these links automatically. 

https://marketplace.visualstudio.com/items?itemName=TylerHarris.terraform-link-docs